client/tests/smokeTest.test.js

```diff
@@ -1,142 +1,88 @@
-const puppeteer = require('puppeteer');
-const path = require('path');
-const spawn = require('cross-spawn');
-const treeKill = require('tree-kill');
+import getPage, { setup, teardown } from './util/getPage';
 
 /*
 End-to-End Testing TODO:
 - The server start and close functions should be handled in one place by
 Jest's setup and teardown scripts.
-- There needs to be a utility function to enable common tasks like getting a
-fresh end-to-end page.
-- End-to-end testing should support signing in with different roles, A.K.A. as
-an admin, tester or vendor.
 - It should be possible to reset the database to a predictable state after each
 test, ideally in such a way that concurrency remains possible. See the POC
 here: https://github.com/w3c/aria-at-app/pull/895
+- Add section to docs for end-to-end testing
```

Comment:
TODO this so it's easier to find afterwards?

Reply:
if you look up at line 4 you can see it's part of a TODO!

Reply:
Right you are. Completely missed that!